src/regexbench.cpp

```diff
@@ -38,22 +38,15 @@ using boost::property_tree::write_json;
 
 namespace po = boost::program_options;
 
-enum EngineType : uint64_t {
-  ENGINE_BOOST,
-  ENGINE_CPP,
-#ifdef HAVE_HYPERSCAN
-  ENGINE_HYPERSCAN,
-#endif
-#ifdef HAVE_PCRE2
-  ENGINE_PCRE2,
-  ENGINE_PCRE2JIT,
-#endif
-#ifdef HAVE_RE2
-  ENGINE_RE2,
-#endif
-#ifdef HAVE_REMATCH
-  ENGINE_REMATCH,
-#endif
+enum class EngineType : uint64_t {
+  boost = 0,
```

Comment:
Is it necessary to set `boost` to 0? If not, I would rather let the compiler choose.